connector/src/main/java/org/geysermc/connector/GeyserConfiguration.java

```diff
@@ -32,7 +32,7 @@
 public interface GeyserConfiguration {
 
     // Modify this when you update the config
-    int CURRENT_CONFIG_VERSION = 3;
+    int CURRENT_CONFIG_VERSION = 4;
```

Comment:
This didn't need to be changed as no new config options where added just a new value and updated comment